src/home/components/SignupContents/EmailAuth/EmailAuth.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+
+import { BoxButton, PlainButton, SimpleTextField } from '@yourssu/design-system-react';
+import { Link } from 'react-router-dom';
+
+import { useSecondTimer } from '@/hooks/useSecondTimer';
+
+import { StyledSignupContentContainer, StyledSignupContentTitle } from '../SignupContents.style';
+
+import {
+  StyledPlainButtonContainer,
+  StyledEmailAuthParagraph,
+  StyledTimerSuffix,
+} from './EmailAuth.style';
+
+interface EmailAuthProps {
+  email: string;
+  onConfirm: () => void;
+}
+
+export const EmailAuth = ({ email, onConfirm }: EmailAuthProps) => {
+  const [authed, setAuthed] = useState(true); // Todo: 인증 기능 구현 완료시 false로 변경
+  const { leftTime, isTimerEnd, resetTimer } = useSecondTimer(8 * 60);
+
+  const leftTimeToString = () => {
```

Comment:
프로젝트 내에서 date-fns 라이브러리를 사용해서 통일하는게 낫지 않을까 싶어요! 
```suggestion
import { format, addSeconds } from 'date-fns';

const leftTimeToString = (leftTime) => {
  const targetTime = addSeconds(new Date(0), leftTime);
  return format(targetTime, 'mm:ss');
};
```

Reply:
오... `date-fns` 를 사용할 생각을 못했네요..
[여기](https://github.com/yourssu/Soomsil-Web/pull/138/commits/0105a5cbf3326df7c98438e8ba429f247ee1180d) 에서 해결했습니다!